core/internal/runmetric/runmetric.go

```diff
@@ -188,9 +187,13 @@ func (mh *MetricHandler) createGlobMetrics(
 // a glob metric, and otherwise returns nil.
 func (mh *MetricHandler) matchGlobMetric(key string) (definedMetric, bool) {
 	for glob, metric := range mh.globMetrics {
-		match, err := filepath.Match(glob, key)
-
-		if err != nil || !match {
+		// filepath doesn't match wildcards if the user has specified a section
+		// using `/` in the logged value. So, we can't use filepath.Match here.
```

Comment:
probably don't need this in the comments, enough to say that we currently only support glob * suffix